src/views/ApartmentApp/index.js

```diff
@@ -182,49 +256,54 @@ export default class ApartApp extends Component {
                       </CardContent>
                     </Card>
                   </Grid>
-                  <Grid item xs={12} md={8} lg={6}>
-                    <Grid container direction="column" spacing={2}>
-                      <Grid item>
-                        <Card>
-                          <ApplicantList
-                            onApplicantRemove={this.onApplicantRemove}
-                            applicants={this.state.applicants}
-                            userProfile={this.state.userProfile}
-                            onSearchSubmit={this.onSearchSubmit}
-                            Authentication={this.props.Authentication}
-                          />
-                          <GordonDialogBox
-                            open={this.state.errorDialogOpen}
-                            onClose={this.handleCloseOkay}
-                            labelledby={'applicant-dialog'}
-                            describedby={'applicant-denied'}
-                            title={'Could Not Add Applicant'}
-                            text={this.state.errorDialogText}
-                            buttonClicked={this.handleCloseOkay}
-                            buttonName={'Okay'}
-                          />
-                        </Card>
-                      </Grid>
-                      <Grid item>
-                        <Card>
-                          <CardHeader title="Preferred Halls" className="card-header" />
-                          <CardContent>
-                            <Typography variant="body1">Placeholder text</Typography>
-                          </CardContent>
-                        </Card>
-                      </Grid>
-                      <Grid item>
-                        <Card>
-                          <CardContent>
-                            <Typography variant="h5">Hello World:</Typography>
+                  <Grid container item xs={12} md={8} lg={6} direction="column" spacing={2}>
+                    <Grid item>
+                      <ApplicantList
+                        applicants={this.state.applicants}
+                        userProfile={this.state.userProfile}
+                        saving={this.state.saving}
+                        savingSuccess={this.state.savingSuccess}
+                        onSearchSubmit={this.onSearchSubmit}
+                        onApplicantRemove={this.onApplicantRemove}
+                        onSaveButtonClick={this.handleSaveButtonClick}
+                        Authentication={this.props.Authentication}
+                      />
+                      {/* <GordonDialogBox
```

Comment:
make sure you get rid of this and its import once you are sure you do not need it.